extensions/amp-form/0.1/amp-form.js

```diff
@@ -155,14 +168,40 @@ export class AmpForm {
     /** @const @private {!./form-validators.FormValidator} */
     this.validator_ = getFormValidator(this.form_);
 
-    // TODO(mkhatib, #6927): Wait for amp-selector to finish loading if the current form
-    // is using it.
-    this.actions_.installActionHandler(
-        this.form_, this.actionHandler_.bind(this));
+    // Wait for amp-selector to finish loading if the current form is using it.
+    this.whenDependenciesReady_().then(() => {
```

Comment:
I'm not sure it makes sense to wait to install action handler here, esp since we have other listeners that could override the behavior. Should, instead, action handler be installed right away, but block on dependencies?